controllers/external-genieacs/cpe-models/base-model.js

```diff
@@ -141,6 +141,7 @@ basicCPEModel.modelPermissions = function() {
       list5ghzChannels: [36, 40, 44, 48, 149, 153, 157, 161, 165],
       allowDiacritics: false, // allows accented chars for ssid and password
       allowSpaces: true, // allows space char for ssid
+      hasWifi: true, // can disable wifi for CPEs that do not support it
```

Comment:
Comentário nonsense, "disable wifi" parece que vai controlar a flag de enable do Wi-Fi
```suggestion
      hasWifi: true, // will collect and display all wifi info
```